submissions/andrewklmn/html-css-popup/app.css

```diff
@@ -0,0 +1,253 @@
+* {
+  padding: 0px;
+  margin: 0px;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  /*
+  background-image: url(img/bg_image.jpg);
+  background-size:     cover;
+  background-repeat:   no-repeat;
+  background-position: center center; 
+  */
+  width: 100vw;  
+  height: 100vh;
+}
+
+label.button {
+  color:black;
```

Comment:
U need reformat all code. In this case need add space after colon.
My preferred css style guide is https://codeguide.academy/html-css.html, u can use any its just example